spring-data-azure-documentdb/src/main/java/com/microsoft/azure/spring/data/documentdb/repository/DocumentDbRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.microsoft.azure.spring.data.documentdb.repository;
+
+import org.springframework.data.repository.NoRepositoryBean;
+import org.springframework.data.repository.PagingAndSortingRepository;
+
+import java.io.Serializable;
+import java.util.List;
+
+@NoRepositoryBean
+public interface DocumentDbRepository<T, ID extends Serializable> extends PagingAndSortingRepository<T, ID> {
+
+    @Override
+    List<T> findAll();
```

Comment:
it's not determined on timeline to support Stream yet. not in scope of this initial checkin.